Update readme and contributing (#590)
pyfar/pyfar: README.rst

.. image:: https://circleci.com/gh/pyfar/pyfar.svg?style=shield
    :target: https://circleci.com/gh/pyfar/pyfar
.. image:: https://mybinder.org/badge_logo.svg
    :target: https://mybinder.org/v2/gh/pyfar/gallery/main?labpath=docs/gallery/interactive/pyfar_introduction.ipynb

The python package for acoustics research (pyfar) offers classes to store
audio data, filters, coordinates, and orientations. It also contains common
functions for digital audio signal processing and plotting audio signals.

Getting Started
===============

> The `pyfar workshop`_ gives an overview of the most important pyfar
> functionality and is a good starting point. It is part of the
> `pyfar example gallery`_ that also contains more specific and in-depth
> examples. The  `pyfar documentation`_ gives a detailed and complete
> overview of pyfar. All these information are available from `pyfar.org`_.

Reviewer: ```suggestion
For a list of application examples including introductions to most important functionality check out the `pyfar example gallery`_ and `pyfar workshop`_ . All gallery example notebooks can be executed interactively without a local installation by clicking the mybinder.org button on the respective example.
A full overview of all features is found in the `pyfar documentation`_ .
```

Author: I actually prefer the current version. It first links the workshop, which might be the easiest entry to the entrire package and explicitly links the gallery, not only pyfar.org. Would you be fine with leaving it as it is?

> The `pyfar workshop`_ gives an overview of the most important pyfar
> functionality and is a good starting point. It is part of the
> `pyfar example gallery`_ that also contains more specific and in-depth
> examples. The  `pyfar documentation`_ gives a detailed and complete
> overview of pyfar. All these information are available from `pyfar.org`_.

Reviewer 2: ```suggestion
The `pyfar workshop`_ gives an overview of the most important pyfar
functionality and is a good starting point. It is part of the
`pyfar example gallery`_ that also contains more specific and in-depth
examples. All gallery example notebooks can be executed interactively without a local installation by clicking the mybinder.org button on the respective example. The  `pyfar documentation`_ gives a detailed and complete
overview of pyfar. All these information are available from `pyfar.org`_.
```
even if we keep it, i would add the info about the interactive notebooks from Marco.

Author: good point - added this information

(Requires Python 3.8 or higher)

Audio file reading/writing is supported through `SoundFile`_, which is based on `libsndfile`_. On Windows and OS X, it will be installed automatically. On Linux, you need to install libsndfile using your distribution’s package manager, for example ``sudo apt-get install libsndfile1``.
If the installation fails, please check out the `help section`_.

Contributing
============

> Check out the `contributing guidelines`_ if you want to become part of pyfar.

Reviewer 2: ```suggestion
Check out the `contributing guidelines`_ if you want to become part of pyfar. For specific guidelines for this packge have a look in the pyfar documentation. 
```
maybe add a link to the local guidelines as well.

Author: This was actually a link to the pyfar specific guidelines. I removed the second sentence to be more clear

.. _pyfar workshop: https://mybinder.org/v2/gh/pyfar/gallery/main?labpath=docs/gallery/interactive/pyfar_introduction.ipynb

> .. _pyfar example gallery: https://pyfar-gallery.readthedocs.io/en/latest/examples_gallery.html

Reviewer: ```suggestion
.. _pyfar example gallery: https://pyfar.org
```

Author: see above

> .. _pyfar documentation: https://pyfar.readthedocs.io/en/stable

Reviewer 2: ```suggestion
.. _pyfar documentation: https://pyfar.readthedocs.io
```

Author: done

.. _pyfar.org: https://pyfar.org

> .. _SoundFile: https://pysoundfile.readthedocs.io/en/stable

Reviewer 2: ```suggestion
.. _SoundFile: https://pysoundfile.readthedocs.io
```

Author: done

.. _libsndfile: http://www.mega-nerd.com/libsndfile/

> .. _help section: https://pyfar-gallery.readthedocs.io/en/latest/help.html

Reviewer 2: ```suggestion
.. _help section: https://pyfar-gallery.readthedocs.io/en/latest/help
```
